package-native.sh

```diff
@@ -41,6 +43,12 @@ while [ $# -gt 0 ]; do
   "--build-id")
     opt_buildid=true
     ;;
+  "--64-only")
+    opt_64_only=1
```

Comment:
Why?

Reply:
To build 64-bit only version of library of course. If build 32 bit version too, library can be built but can't be linked with SDL because FreeBSD package manager can install only 64 or 32 bit, so need to build SDL2 for 32 bit too.

Reply:
Let's use dependency(...) then for Linux/BSD paths.

I am okay with not caring about 32-bit BSDs, it's 2024.